src/harness/reference_models/dpa/dpa_mgr.py

```diff
@@ -432,6 +451,70 @@ def CheckInterference(self, sas_uut_active_grants, margin_db,
 
     return max_diff_interf <= margin_db
 
+  def __PrintStatistics(self, results, margin_db, dpa_name, channel):
+    timestamp = datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+    filename='%s DPA=%s channel=%s.csv' % (timestamp, self.name, channel)
```

Comment:
Both "dpa_name" are "self.name" are used in the function. For consistency, it could  be better to use just one of them.